### DISCUSSION TITLE
我的大学四年 | 周鑫的个人博客

### DISCUSSION THREAD

**Discussion by giscus** [LittleHeroZZZX/zhouxin-space#62] [Announcements]
# 我的大学四年 | 周鑫的个人博客

前言 时间过得真是快呀，转眼本科就要毕业了，切身体会到了白驹过隙的感觉：人生天地之间，若白驹之过隙，忽然而已。最近刚把博客搭得有点模样，趁着热

https://www.zhouxin.space/thoughts/reflections-on-my-university-years/

**Comment by KyleCaiW** [comment]
文风很好，颇有江苏高考遗风，根据时间推断，也是最后一届08方案了吧。我也是，但和博主专业大相径庭，很认同博主的观点，进入大学盲目地卷其实收益不高，庆幸博主能一路相遇贵人，可以毫不心虚地说没有荒废四年。这是我至今仍后悔的地方，没有引路人，没有对未来的规划，只能硬卷，终究只能混个中上。感谢博主分享，祝愿博主万事顺遂。

**Reply by LittleHeroZZZX** [reply]
是嘚，最后一届 08 方案，我更擅长副科，对这个方案一直有点意难平。最近在准备实习和秋招，陷入了迷茫和焦虑，收到评论又看了遍博客，想起来大学四年也是这样关关难过关关过的，心安不少。

之前考研的时候别人经验贴里的一句话：
> 那些遥远而大的东西，模糊而令人紧张，眼前每一道题目清晰而又熟悉。思在当下，年少不可轻负，虑于长足，久恨皆于自误。

共勉～
